inference/core/interfaces/camera/video_source.py

```diff
@@ -1083,7 +1092,8 @@ def decode_video_frame_to_buffer(
     buffer: Queue,
     decoding_pace_monitor: sv.FPSMonitor,
     source_id: Optional[int],
-    fps: float = 0,
+    fps: Optional[float] = 0,
```

Comment:
Maybe we should make it `None` as default since it`s marked as optional?

Reply:
👍